seed/api/v3/urls.py

```diff
@@ -107,6 +108,9 @@
 properties_router.register(r'meters', MeterViewSet, basename='property-meters')
 properties_router.register(r'notes', NoteViewSet, basename='property-notes')
 properties_router.register(r'scenarios', PropertyScenarioViewSet, basename='property-scenarios')
+# This is a third level router, so we need to register it with the second level router
+meters_router = nested_routers.NestedSimpleRouter(properties_router, r'meters', lookup='meter')
```

Comment:
@anchapin  -- here is the example of the third level route which will also need to be created for /properties/1/scenarios/10/measures